LedgerWallet/Transports/ILedgerTransport.cs

```diff
@@ -8,6 +8,6 @@ namespace LedgerWallet.Transports
 {
 	public interface ILedgerTransport
 	{
-		Task<byte[][]> Exchange(byte[][] apdus);
+		Task<byte[][]> ExchangeAsync(byte[][] apdus);
```

Comment:
Can you remove this change? it is breaking existing code for no reason.

Reply:
No problem.

Are you sure? It's a Microsoft convention to suffix all task based method with Async. When I look at the code, and I see a method without Async, I assume it's synchronous.